public/js/controllers/game.js

```diff
@@ -30,97 +28,94 @@ angular.module('mean.system')
       }
     };
 
-    $scope.pointerCursorStyle = function () {
+    $scope.pointerCursorStyle = () => {
       if ($scope.isCzar() && $scope.game.state === 'waiting for czar to decide') {
-        return { cursor: 'pointer' };
+        return {
+          cursor: 'pointer'
+        };
       }
       return {};
     };
 
-    $scope.sendPickedCards = function () {
+    $scope.sendPickedCards = () => {
```

Comment:
Is there a reason for omitting documentation for other functions in your PR

Reply:
i didn't add those other functions i only wrote documentation for methods that i added like drawCard method and setCzar method